### PR TITLE
Using Environment Variables for Backend Services

### PR DESCRIPTION
**Pull request by shaikh-rashid**

### CODE REVIEW

File: src/constants/constants.js [hunk 1]
```diff
@@ -3,11 +3,11 @@
  */
 
 // API URL where the Dashboard get all the data
-const BASE_URL = 'http://ga4ghdev01.bcgsc.ca:20127';
+const BASE_URL = process.env.REACT_APP_BASE_URL;
```
Comment (fcoralsasso) [inline src/constants/constants.js:6]: Nice.

**Review by fcoralsasso — APPROVED** [review]
lgtm

**Review by dashaylan — APPROVED** [review]

**Comment by haoyuanli** [comment]
can we call the variable something else other than `REACT_APP`?

**Comment by shaikh-rashid** [comment]
> can we call the variable something else other than `REACT_APP`?
This is the convention set by React for environment variables
https://create-react-app.dev/docs/adding-custom-environment-variables/

**Pull request merged**